examples/filtered_tracks.py

```diff
@@ -0,0 +1,82 @@
+import discord
+import wavelink
+from discord.ext import commands
+
+class Bot(commands.Bot):
+    def __init__(self, *args, **kwargs):
+        super(Bot, self).__init__(*args, **kwargs)
+    
+    async def on_ready(self):
+        print("-----")
+        print(f"Logged in as: {self.user.name}#{self.user.discriminator}")
+        print(f"ID: {self.user.id}")
+        print("-----")
+
+class FilteredMusic(commands.Cog):
+    """Music cog to hold Wavelink related commands and listeners."""
+
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+
+        bot.loop.create_task(self.connect_nodes())
+
+        self.players = {}
```

Comment:
You don't need to do this. Your player is stored in state throughout discord.py.
E.g context/guild.voice_client